jekyll-assets/css/style.css

```diff
@@ -1,35 +1,71 @@
 :root {
-  --red: #cd2355;
+  --bg: #000000;
+  --footer-bg: #050609;
+  --near-bg: #111;
+  --top-title-colour: #343434;
   --red-tint: #d75a64;
-  --docsearch-primary-color: var(--red);
+  --brand-colour: #cd2355;
+  --copy-button-colour: #d8d8d8;
+  --code-bg-colour: #151515;
+  --toc-hover-colour: #252525;
+  --subtle: #707070;
+  --less-subtle: #333;
+  --home: #cd2355;
+  --rp2040-context-tag: #50C878;
```

Comment:
Does there also need to be a `--rp2350-context-tag` ? :shrug:

Reply:
I'll take a look at the SDK doc today to check, great point. There probably SHOULD be!